biome.json

```diff
@@ -0,0 +1,35 @@
+{
+	"$schema": "https://biomejs.dev/schemas/1.9.3/schema.json",
+	"vcs": {
+		"enabled": false,
+		"clientKind": "git",
+		"useIgnoreFile": true
+	},
+	"files": {
+		"ignoreUnknown": false,
+		"ignore": [],
+		"include": ["./app/javascript/**/*.js"]
+	},
+	"formatter": {
+		"enabled": true,
+		"indentStyle": "tab",
+		"useEditorconfig": true
```

Comment:
Currently, our `.editorconfig` has:

```
indent_style = space
indent_size = 2
```

Will this conflict since it's using tabs?

Reply:
good point, indentStyle overrides the imported editorconfig. I removed it, so its consistent across all files.